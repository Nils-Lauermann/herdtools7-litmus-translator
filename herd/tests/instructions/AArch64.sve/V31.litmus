AArch64 V31
{
uint64_t 0:X3;
0:X0=0; 0:X1=1;
}
   P0               ;
 CMP W0,W1          ;
 WHILELT P0.S,W0,W1 ;
 MRS X3,NZCV        ;

forall 0:X3=0xa0000000