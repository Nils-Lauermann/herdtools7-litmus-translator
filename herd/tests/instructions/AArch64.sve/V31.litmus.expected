Test V31 Required
States 1
0:X3=2684354560;
Ok
Witnesses
Positive: 1 Negative: 0
Flag Scalable-Vector-Extensions-is-work-in-progress
Condition forall (0:X3=2684354560)
Observation V31 Always 1 0
Hash=8c2fa76386c9fbda4a6bbe9961e4dbed

